src/shapes/Transformer.ts

```diff
@@ -1745,8 +1745,6 @@ Factory.addGetterSetter(Transformer, 'ignoreStroke', false);
  */
 Factory.addGetterSetter(Transformer, 'padding', 0, getNumberValidator());
 
-Factory.addGetterSetter(Transformer, 'node');
-
```

Comment:
I think this was added accidentally - `Transformer` never references `this.node` as far as I can see...

Reply:
This is an old (deprecated) API. Should be safe to remove.